src/app/ui/components/containers/dialog/dialog.tsx

```diff
@@ -68,16 +76,20 @@ export function Dialog({ children, footer, header, onClose, isShowing }: RadixDi
           >
             {header && cloneElement(header, { onClose })}
 
-            <Box
-              style={{
-                height: '100%',
-                maxHeight: contentMaxHeight,
-                marginBottom: footer ? token('sizes.footerHeight') : token('spacing.space.04'),
-                overflowY: 'auto',
-              }}
-            >
-              {children}
-            </Box>
+            {wrapChildren ? (
```

Comment:
All other dialogs have this `Box` to help them size dynamically but ones containing `Virtuoso` lists now bypass that.